packages/react-core/src/components/Menu/MenuItem.tsx

```diff
@@ -22,6 +24,8 @@ export interface MenuItemProps extends Omit<React.HTMLProps<HTMLLIElement>, 'onC
   itemId?: any;
   /** Target navigation link */
   to?: string;
+  /** @beta Flag indicating the item has a checkbox */
+  isCheck?: boolean;
```

Comment:
Would `hasCheck` make more sense here? Not a blocker as I'm happy with either prop name.

Reply:
I was waiting for someone to comment.  I was debating on the name here.